### PR TITLE
Add MuJoCo kitchen sim to ur workspace

### PR DESCRIPTION
**Pull request by MikeWrock**

### CODE REVIEW

File: src/picknik_ur_mujoco_config/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# picknik_ur_mujoco_config
+
+A MoveIt Pro base configuration for PickNik's Universal Robots (UR) arms.
```
Comment (MikeWrock) [inline src/picknik_ur_mujoco_config/README.md:3]:
```suggestion
A MoveIt Pro MuJoCo simulation for PickNik's Universal Robots (UR) arms.
```

File: src/picknik_ur_mujoco_config/description/picknik_ur_attachments_macro.xacro
Comment (MikeWrock) [inline src/picknik_ur_mujoco_config/description/picknik_ur_attachments_macro.xacro:1]: If there is an easier way to get the robotiq gripper without the ros2_control tags, I would prefer a different approach here

File: src/picknik_ur_mujoco_config/objectives/take_snap.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root BTCPP_format="4" main_tree_to_execute="take snap">
```
Comment (MikeWrock) [inline src/picknik_ur_mujoco_config/objectives/take_snap.xml:2]:
```suggestion
<root BTCPP_format="4" main_tree_to_execute="Take snapshot">
```

File: src/picknik_ur_mujoco_config/objectives/take_snap.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  <!--//////////-->
+  <BehaviorTree ID="take snap" _description="" _favorite="true">
```
Comment (MikeWrock) [inline src/picknik_ur_mujoco_config/objectives/take_snap.xml:4]:
```suggestion
  <BehaviorTree ID="Take snapshot" _description="" _favorite="true">
```

File: src/picknik_ur_mujoco_config/objectives/take_snap.xml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    <Control ID="Sequence" name="TopLevelSequence">
+      <Action ID="ClearSnapshot"/>
+      <Action ID="GetPointCloud" topic_name="/scene_camera/points"/>
+      <Action ID="BetterSendpointcloudtoui" sensor_name="world"/>
+    </Control>
+  </BehaviorTree>
+  <TreeNodesModel>
+    <SubTree ID="take snap"/>
```
Comment (MikeWrock) [inline src/picknik_ur_mujoco_config/objectives/take_snap.xml:12]:
```suggestion
    <SubTree ID="Take snapshot"/>
```

File: src/picknik_ur_mujoco_config/objectives/wrist_snap.xml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+<root BTCPP_format="4" main_tree_to_execute="wrist snap">
+  <!--//////////-->
+  <BehaviorTree ID="wrist snap" _description="" _favorite="true">
+    <Control ID="Sequence" name="TopLevelSequence">
+      <Action ID="GetPointCloud" topic_name="/wrist_camera/points"/>
+      <Action ID="BetterSendpointcloudtoui" sensor_name="world"/>
+    </Control>
+  </BehaviorTree>
+  <TreeNodesModel>
+    <SubTree ID="wrist snap"/>
```
Comment (MikeWrock) [inline src/picknik_ur_mujoco_config/objectives/wrist_snap.xml:11]:
```suggestion
<root BTCPP_format="4" main_tree_to_execute="Wrist camera snapshot">
  <!--//////////-->
  <BehaviorTree ID="Wrist camera snapshot" _description="" _favorite="true">
    <Control ID="Sequence" name="TopLevelSequence">
      <Action ID="GetPointCloud" topic_name="/wrist_camera/points"/>
      <Action ID="BetterSendpointcloudtoui" sensor_name="world"/>
    </Control>
  </BehaviorTree>
  <TreeNodesModel>
    <SubTree ID="Wrist camera snapshot"/>
```

File: src/picknik_ur_mujoco_config/package.xml [hunk 6]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>picknik_ur_mujoco_config</name>
+  <version>5.1.0</version>
+
+  <description>Base configuration package for Picknik's UR robot arms</description>
```
Comment (MikeWrock) [inline src/picknik_ur_mujoco_config/package.xml:6]:
```suggestion
  <description>MuJoCo simulation configuration package for Picknik's UR robot arm in a kitchen</description>
```

File: src/picknik_ur_mujoco_config/objectives/take_snap.xml [hunk 7]
```diff
@@ -0,0 +1,14 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root BTCPP_format="4" main_tree_to_execute="Take snapshot">
+  <!--//////////-->
+  <BehaviorTree ID="Take snapshot" _description="" _favorite="true">
+    <Control ID="Sequence" name="TopLevelSequence">
+      <Action ID="ClearSnapshot"/>
+      <Action ID="GetPointCloud" topic_name="/scene_camera/points"/>
+      <Action ID="BetterSendpointcloudtoui" sensor_name="world"/>
```
Comment (pac48) [inline src/picknik_ur_mujoco_config/objectives/take_snap.xml:8]: I don't have this behavior. Is it custom?

**Comment by MikeWrock** [comment]
We don't plan on moving forward with the kitchen config

**Pull request closed without merging**